File: python/Day03/demo02.py
#递归梅开二度
def fibonacci(num):
    if num == 1:
        return 1
    elif num == 2:
        return 1
    elif num == 3:
        return 2
    else:
        return fibonacci(num - 1) + fibonacci(num - 2)
print(fibonacci(12))

File: Pratice/src/test/meng/demo16Inner/Demo02.java
package test.meng.demo16Inner;

public class Demo02 {
    public int num = 10;
    public class Inner{
        int num = 20;
        public void Method(){
            int num = 30;
            System.out.println(num);
            System.out.println(this.num);
            System.out.println(Demo02.this.num);
        }
    }
}
